src/Grid/grid.jl

```diff
@@ -131,7 +180,22 @@ end
 
 n_faces_per_cell(grid::Grid) = nfaces(eltype(grid.cells))
 
+"""
+    function compute_vertex_values(grid::AbstractGrid, f::Function)
+    function compute_vertex_values(grid::AbstractGrid, v::Vector{Int}, f::Function)    
+    function compute_vertex_values(grid::AbstractGrid, set::String, f::Function)
+
+Given a `grid` and some function `f`, `compute_vertex_values` computes all nodal values,
```

Comment:
Can we change the name to `compute_nodal_values` now or would this be breaking already?